test/AutoRest.TestServerLowLevel.Tests/dpg_initial.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using AutoRest.TestServer.Tests.Infrastructure;
+using dpg_initial_LowLevel;
+using NUnit.Framework;
+
+namespace AutoRest.TestServer.Tests
+{
+    public class DpgInitialTest : TestServerLowLevelTestBase
+    {
+        [Test]
+        public Task DpgAddOptionalInput_NoParams_Test() => Test(async (host) =>
```

Comment:
What are the tests validating in dpg_initial.cs?  Is it establishing a baseline so we know the same code works on both versions?  Because in this method, we are not validating that you can add an optional parameter -- that doesn't happen until the v2 client in dpg_update1.cs.

My goal for this comment is just to make sure the test is named in a way that other developers can understand.  If we want to keep the name AddOptionalInput, let's add a comment stating where the addition happens so folks aren't confused when it's not here.

Reply:
The test name has to start with DpgAddOptionalInput_NoParams which matches the name in testserver, otherwise we get a coverage report error, but we can modify the suffix after the last `_`, maybe use V1 and V2 correspondingly. I will add comments as well.